EIPS/eip-3779.md

```diff
@@ -8,6 +8,7 @@ status: Withdrawn
 type: Standards Track
 category: Core
 created: 2021-08-30
+requires: 2315, 3540, 3670, 3779, 4200
```

Comment:
CC @SamWilsn - `preamble-requires-status` shouldn't trigger when the status is withdrawn